kafka.cpp

```diff
@@ -380,15 +383,26 @@ Kafka::send(const vector<Reading *> readings)
 		payload << "{ \"asset\" : " << quote(assetName) << ", ";
 		payload << "\"timestamp\" : " << quote((*it)->getAssetDateUserTime(Reading::FMT_ISO8601MS, true)) << ", ";
 		vector<Datapoint *> datapoints = (*it)->getReadingData();
+		bool isImage = true;
 		for (auto dit = datapoints.cbegin(); dit != datapoints.cend();
 					++dit)
 		{
+			DatapointValue dpv = (*dit)->getData();
+			DatapointValue::dataTagType dataType = dpv.getType();
+			if ( dataType == DatapointValue::T_IMAGE || dataType == DatapointValue::T_DATABUFFER )
+			{
+				// SKIP Image data type
+				Logger::getLogger()->info("Image and databuffer are not supported. Datapoint %s of asset %s has image/databuffer",assetName.c_str(), (*dit)->getName().c_str());
```

Comment:
Are DP and asset names swapped here in the last 2 params?

Reply:
Fixed